unlock-app/src/components/creator/lock/EmbedCodeSnippet.js

```diff
@@ -10,9 +10,9 @@ export function EmbedCodeSnippet({ lock }) {
 <script src="https://unlock-protocol.com/unlock.js"></script>
 
 <!-- Lock elements by wrapping them in this div -->
-<div unlock-lock="${lock.address}">
+<div unlock-lock="${lock.transaction}">
```

Comment:
This is also probably not the right thing. The lock.transaction value is actually the hash of the transaction, I do not think we can replace this.